app/models/piece.rb

```diff
@@ -3,11 +3,33 @@ class Piece < ActiveRecord::Base
 	belongs_to :game
 
 
-	#Allows an easier navigation between model. E.g.  Rook.find(56) => <Rook id: 56, name: :rook, color: :white, image: nil, position: 00, game_id: 20...>
+  #Allows an easier navigation between model. E.g.  Rook.find(56) => <Rook id: 56, name: :rook, color: :white, image: nil, position: 00, game_id: 20...>
   scope :rooks, -> { where(type: 'Rook') } 
   scope :knights, -> { where(type: 'Knight') } 
   scope :bishops, -> { where(type: 'Bishop') }
   scope :kings, -> {where(type: 'King') }
   scope :queens, -> {where(type: 'Queen') }
   scope :pawns, -> {where(type: 'Pawn') }
+
+  def move_to!(x_axis, y_axis)#Check methods .same_piece() and .capture()
```

Comment:
let's get some unit tests for the `move_to!` method.  We should test all three cases:

* Move to an empty spot
* Move to an enemy spot
* Move to allied spot (traitor!)